lib/src/native/media_stream_track_impl.dart

```diff
@@ -15,11 +17,14 @@ class MediaStreamTrackNative extends MediaStreamTrack {
   final String _kind;
   bool _enabled;
 
+  bool _muted = false;
+
   @override
   set enabled(bool enabled) {
     _channel.invokeMethod('mediaStreamTrackSetEnable',
         <String, dynamic>{'trackId': _trackId, 'enabled': enabled});
     _enabled = enabled;
+    _muted = enabled;
```

Comment:
I think it should be ` _muted = !enabled`;

Reply:
You're right